RecoEgamma/EgammaHLTProducers/BuildFile.xml

```diff
@@ -1,29 +0,0 @@
-<use   name="FWCore/Framework"/>
```

Comment:
This Buildfile was just emptied, not really removed: please ```git rm``` now the empty Buildfile

Reply:
For me, `scram` did not descend into the plugins directory without seeing a BuildFile here. I can try again with a clean environment.

Reply:
Ok of course you were right, it works this time...